app/assets/v2/js/cart.js

```diff
@@ -1003,6 +973,33 @@ Vue.component('grants-cart', {
       indicateMetamaskPopup(true);
     },
 
+
+    /**
+     * @notice Returns all tokens available on the selected network allowing for isPolygon override
+     */
+    getTokens(_filter = false, isPolygon = false) {
+      // return a list of uniq (by name) tokens
+      const uniq = {};
+
+      // get the networkId
+      const networkId = (
+        isPolygon ? (
+          this.network === 'mainnet' ? POLYGON_MAINNET_NETWORK_ID : POLYGON_TESTNET_NETWORK_ID
+        ) : (
+          // rinkeby is stored as networkId==1 in the db
+          this.network === 'rinkeby' ? 1 : this.networkId
+        )
+      );
```

Comment:
why was rinkeby networkId changed from 1 to 4 in db? isn't that defeating the purpose of networkId as relates to chainId ?

Reply:
I don't think it was ever entered as 4 - have you checked the tokens on prod/staging?

Reply:
hmmm... i've always had it set to 4 locally... not sure why prod is different